tests/Microsoft.Bot.Builder.TestBot.Json/Samples/GeneratedForm/luis/sandwich.en-us.lu.dialog

```diff
@@ -1,11 +1,6 @@
-
 {
-    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/sdk.schema",
-    "$kind": "Microsoft.LuisRecognizer",
+    "$type": "Microsoft.LuisRecognizer",
     "applicationId": "=settings.luis.sandwich_en-us_lu",
```

Comment:
You should not have changed any of this.  Should be $kind and should include predictionOptions.

Reply:
oh I ran bf luis:build and it clobbered this.  You need to update luis build

---
In reply to: [376592062](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376592062) [](ancestors = 376592062)